File: src/main/java/com/davidm1a2/afraidofthedark/common/constants/ModRecipes.java
package com.davidm1a2.afraidofthedark.common.constants;

import com.davidm1a2.afraidofthedark.common.recipe.ScrollCombineRecipe;
import net.minecraft.item.crafting.IRecipe;

/**
 * Class containing a list of custom mod recipes
 */
public class ModRecipes
{
    // The recipe used to combine two or more recipes
    public static final IRecipe SCROLL_COMBINE_RECIPE = new ScrollCombineRecipe();

    // A list of mod recipes
    public static final IRecipe[] MOD_RECIPES = new IRecipe[]
            {
                    SCROLL_COMBINE_RECIPE
            };
}

File: src/main/java/com/davidm1a2/afraidofthedark/common/constants/ModRecipes.kt
package com.davidm1a2.afraidofthedark.common.constants

import com.davidm1a2.afraidofthedark.common.recipe.ScrollCombineRecipe
import net.minecraft.item.crafting.IRecipe

/**
 * Class containing a list of custom mod recipes
 */
object ModRecipes
{
    // The recipe used to combine two or more recipes
    val SCROLL_COMBINE_RECIPE = ScrollCombineRecipe()

    // A list of mod recipes
    val MOD_RECIPES = arrayOf(
            SCROLL_COMBINE_RECIPE
    )
}